master/buildbot/status/slave.py

```diff
@@ -187,6 +188,8 @@ def asDict(self, include_build_steps=True, include_build_props=True):
         result['runningBuilds'] = builds
         result['lastMessage'] = self.lastMessageReceived()
         result['health'] = self.health
+        result['fqdn'] = self.fqdn
+        result['managerUrl'] = self.master.config.slaveManagerUrl
```

Comment:
@BarryUnity I've just notice that in json we are using 'managerUrl' instead of slaveManagerUrl ? could you change it to 'slaveManagerUrl'